doc/manual/source_localization/forward.rst

```diff
@@ -251,143 +251,6 @@ with positive direction up.
 
 .. note:: The above definition is identical to that    of the Neuromag MEG/EEG (head) coordinate system. However, in 4-D    Neuroimaging and CTF MEG systems the head coordinate frame definition    is different. The origin of the coordinate system is at the midpoint    of the left and right auricular points. The :math:`x` axis    passes through the nasion and the origin with positive direction    to the front. The :math:`y` axis is perpendicular    to the :math:`x` axis on the and lies in    the plane defined by the three fiducial landmarks, positive direction    from right to left. The :math:`z` axis is    normal to the plane of the landmarks, pointing up. Note that in    this convention the auricular points are not necessarily located    on :math:`y` coordinate axis. The file conversion utilities take care of these idiosyncrasies and convert all coordinate information    to the MNE software head coordinate frame.
 
-.. _BEHCGJDD:
-
-Creating a surface-based source space
-#####################################
-
-The fif format source space files containing the dipole locations
-and orientations are created with the utility :ref:`mne_make_source_space`.
-This utility is usually invoked by the convenience script :ref:`mne_setup_source_space`,
-see :ref:`setting_up_source_space`.
-
-
-.. _BJEFEHJI:
-
-Creating a volumetric or discrete source space
-##############################################
-
-In addition to source spaces confined to a surface, the MNE
-software provides some support for three-dimensional source spaces
-bounded by a surface as well as source spaces comprised of discrete,
-arbitrarily located source points. The :ref:`mne_volume_source_space` utility
-assists in generating such source spaces.
-
-
-.. _BEHCACCJ:
-
-Creating the BEM meshes
-#######################
-
-The :ref:`mne_surf2bem` utility
-converts surface triangle meshes from ASCII and FreeSurfer binary
-file formats to the fif format. The resulting fiff file also contains
-conductivity information so that it can be employed in the BEM calculations.
-See command-line options in :ref:`mne_surf2bem`.
-
-.. note:: The utility ``mne_tri2fiff`` previously used for this task has been replaced by :ref:`mne_surf2bem`.
-
-.. note:: The convenience script :ref:`mne_setup_forward_model` described in :ref:`CIHDBFEG` calls :ref:`mne_surf2bem` with the appropriate options.
-
-.. note:: The vertices of all surfaces should be given in the MRI coordinate system.
-
-
-.. _BEHDEFCD:
-
-Tessellation file format
```

Comment:
the `tessellation file format` might be of interest. (However, no one was pointing at it) maybe we should make an `:orphan:` page and let (this_missing_ci_link) and (this_other_missing_ci_link) point to it. What I don't know is what is the extension of this tessellation file. Is it `-trans.fif` `-head.fif` ?? And why are 3D mesh files `.fif`?

Reply:
Yes it's .fif. We probably don't need to talk about it since it's implicitly in the code if people need to write a reader for it. Meshes can be stored in FIF for I/O convenience

Reply:
I think it's just a text format that the MNE code could read to produce .surf or .fif binary files.

I copy this into a mne-c specific doc file. It's clearly not used by mne-python

Reply:
So no need to add it. If someone asks for it, for whatever reason then we do add it.